src/core/crypto.c

```diff
@@ -1675,6 +1675,13 @@ QuicCryptoProcessDataComplete(
     _In_ uint32_t RecvBufferConsumed
     )
 {
+    if (Crypto->CertValidationPending || Crypto->TicketValidationStatus == QUIC_SERVER_VALIDATION_PENDING) {
+        if (Crypto->TicketValidationStatus == QUIC_SERVER_VALIDATION_PENDING) {
+            Crypto->PendingValidationBufferLength = RecvBufferConsumed;
```

Comment:
You shouldn't only cache this for ticket validation pending. Do for cert pending too.